effekt/jvm/src/main/scala/effekt/Driver.scala

```diff
@@ -108,6 +109,58 @@ trait Driver extends kiama.util.Compiler[EffektConfig, EffektError] { outer =>
     afterCompilation(source, config)(context)
   }
 
+  def generateCrashReport(e: Throwable, context: Context, config: EffektConfig): Unit = {
+    import java.nio.file.{Files, Paths}
+    import java.util.Properties
+
+    // Capture general information
+    val effektVersion = "Effekt Compiler Version: " // Null:  + System.getProperty("effekt.version")
```

Comment:
See here for how we normally get the Effekt version:

https://github.com/effekt-lang/effekt/blob/82af20f4f356f63df61cc7f610af5c37b5bc99d7/effekt/jvm/src/main/scala/effekt/EffektConfig.scala#L9-L10